app.js

```diff
@@ -1,60 +1,73 @@
-const harcodeJSON = ``
-
 const bodyParser = require('body-parser')
 const express = require('express')
 const app = express()
 
+const stations = require('./middleware/stations');
+
+
 
 // configure app to use bodyParser()
 // this will let us get the data from a POST
-app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.urlencoded({
+  extended: true
+}));
 app.use(bodyParser.json());
 
 app.get('/', (req, res) => {
-    res.send('Hello World!')
+  res.send('Hello World!')
+})
+
+app.get('/api/stations', (req, res) => {
+  console.log('fetch stations from original API');
+  stations.fetchStations()
+    .then((data) => {
+      res.status(200)
+      res.set('Content-Type', 'application/json')
```

Comment:
https://stackoverflow.com/questions/19696240/proper-way-to-return-json-using-node-or-express/23198978#23198978